packages/mobile/src/screens/SingleMovie2.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import {FlatList} from 'react-native';
+//Images
+import player from '../../assets/img/player.jpg';
+import back_arrow from '../../assets/img/back_arrow2.png';
+import h from '../../assets/img/h.png';
```

Comment:
Zdjęcia do projektu powinny pochodzić z zasobów natywnych. Tak jak to zrobiła justyna. Dlatego że telefony natywnie ogarniaja sobie różnej wielkości zdjęcia w zależności od wielkosci ekranu.